libs/chiliui/chili/headers/util.lua

```diff
@@ -336,6 +319,18 @@ function table:shallowcopy()
 	return newTable
 end
 
+function table:deepcopy()
```

Comment:
deepcopy is faster than merge when there is nothing left to merge, as it avoids lookups on the target table.